Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM wisvch/debian:stretch
+MAINTAINER Mark Janssen <mark@praseodym.net>
+
+RUN apt-get update && apt-get install -y nodejs nodejs-legacy && rm -rf /var/lib/apt/lists/*
+COPY florriebot.tar.gz /tmp
```

Comment:
Aah okay. But then this Dockerfile won't work "standalone", i.e. when a user checks out the repo and runs `docker build`. Is that desirable?

-Rick

> On 11 Sep 2016, at 12:25, Mark Janssen <notifications@github.com> wrote:
> 
> In Dockerfile:
> 
> > @@ -0,0 +1,15 @@
> > +FROM wisvch/debian:stretch
> > +MAINTAINER Mark Janssen <mark@praseodym.net>
> > +
> > +RUN apt-get update && apt-get install -y nodejs nodejs-legacy && rm -rf /var/lib/apt/lists/*
> > +COPY florriebot.tar.gz /tmp
> TeamCity is doing this :)
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>